rollup.config.js

```diff
@@ -40,7 +40,8 @@ function rollupConfig({ formats, prod }) {
             format,
             banner,
             footer,
-            file: path.join(targetDirectory, targetName)
+            file: path.join(targetDirectory, targetName),
+            exports: 'auto'
```

Comment:
Does not seem to change the output file format, and suppresses a warning from Rollup.